tests/backend/test_tf_ops.py

```diff
@@ -51,3 +51,25 @@ def test_reduced_density_matrix_fock_states(self, num_modes, setup_backend, cuto
         expected = tf.constant(expected)
 
         assert np.allclose(reduced_dm, expected, atol=tol, rtol=0)
+
+    @pytest.mark.parametrize("modes", [[0], [1], [2], [0, 1], [0, 2], [1, 2], [0, 1, 2]])
+    def test_reduced_density_matrix_multiple_modes(self, setup_backend, cutoff, modes, tol):
+        """Test the reduced_density_matrix returns the correct reduced density matrices."""
+        state = np.zeros([cutoff, cutoff, cutoff])
+        state[0, 0, 0] = 1 / np.sqrt(2)
+        state[1, 1, 1] = 1 / np.sqrt(2)
+        state = np.multiply.outer(state, state.conj())
+
+        # transpose the state due to different conventions
```

Comment:
What are the different conventions? We should only need to use SF convention here :thinking:

Reply:
I think it's the way [`np.multiply.outer`](https://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.ufunc.outer.html) works, by ordering the indices, is different from what's done in SF. I could be wrong though, but I remember this being the case in TW at least. :thinking:

Reply:
ahh yes, got it thanks :smiley: